src/components/app-configure/CreateApp.vue

```diff
@@ -49,6 +49,7 @@ async function handleCreateApp() {
       region: RegionMapping[selectedRegion.value.value],
     })
   ).data.app
+  const appOverview = (await fetchStats(app.ID)).data
```

Comment:
looks like if `fetchStats` throws error, the app might crash

Reply:
Good point. Wrapped the entire flow in try catch and showing toast if error occurs. Resolved [here](https://github.com/arcana-network/developer-dashboard/pull/135/commits/7ace58432fb05f1afdc58d173ea1be8c9918b9ab)